staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3046,6 +3046,17 @@ type PodSpec struct {
 	// +listMapKey=topologyKey
 	// +listMapKey=whenUnsatisfiable
 	TopologySpreadConstraints []TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty" patchStrategy:"merge" patchMergeKey:"topologyKey" protobuf:"bytes,33,opt,name=topologySpreadConstraints"`
+	// TTLSecondsAfterFinished limits the lifetime of a Pod that has finished
```

Comment:
this also needs to be added to the "Internal" types.go file and conversion regenerated